conclusion_and_future_work.tex

```diff
@@ -0,0 +1,33 @@
+SymPy is a robust CAS that provides a wide array of features. It is written in
+a general purpose programming language, Python, which allows it to be used in
+a first-class way with other Python projects, including the scientific Python
+stack. It is designed to be used in an extensible way. Unlike many other CASs,
+it is designed to be used both as a end-user application and as a library.
+
+SymPy expressions are built from immutable trees of Python classes. It uses
+Python both as the internal language and the user language, meaning users can
+use the same methods that the library implements to extend it. SymPy has an
+assumptions system for declaring and deducing mathematical properties on
+expressions.
+
+The numerics of SymPy are implemented in the mpmath library, which uses
+arbitrary precision floating point arithmetic implemented in pure Python. This
+allows expressions to be evaluated with concrete data as needed.
+
+SymPy has submodules for many areas of mathematics. It has functions for
+simplifying expressions, doing common calculus operations, pretty printing
+expressions, solving equations, and symbolic matrices. Other areas also
+included are discrete math, concrete math, plotting, geometry, statistics,
+polynomials, sets, series, vectors, combinatorics, group theory, code
+generation, tensors, Lie algebras, cryptography, and special functions.
+Additionally, SymPy contains submodules targeting certain specific domains,
+such as classical mechanics and quantum mechanics.
+
+% Future work:
+
+Some of the planned future work for SymPy includes work on improving code
+generation, improvements to the speed of SymPy, and improving the solvers
+module.
+% Feel free to add stuff here.
+
+% TODO: Mention SymEngine.
```

Comment:
@certik do you want to write anything about SymEngine to the future work section of the conclusion?